docs/stake-pool-course/handbook/utxo-model.md

```diff
@@ -11,11 +11,11 @@ In the classic UTxO model (Cardano SL in Byron and Shelley), a transaction outpu
 * its value (the amount)
 * its reference address (the "proof" of ownership telling who is the owner of the output)
 
-Every transaction has at least one input and at least one output. Transactions carries the information about money flow. Inputs inform where the money came from (source address), while outputs inform where the money goes to (destination address). Every new transactions spend outputs of previous transactions and produce new outputs that can be spent by future transactions. The blockchain records the collective history of those transactions. 
+Every transaction has at least one input and at least one output. Transactions carry information about money flow. Inputs indicate where the money came from (source address), while outputs indicate where the money goes to (destination address). Every new transaction spends outputs of previous transactions and produces new outputs that can be spent by future transactions. The blockchain records the collective history of these transactions.
 
-An unspent transaction output is called a UTxO (as in **U**nspent **Tx** **O**utput) and represents an amount of money owned by a participant that can be spent as an input in a new transaction. The key issue here is that a complete UTxO must be used as an input for a new transaction. UTxOs cannot be consumed in part. Rather, the difference is sent back to the source as a "change" during the transaction, which is represented as a new UTxO. 
+An unspent transaction output is called a UTxO (as in **U**nspent **Tx** **O**utput) and represents an amount of money owned by a participant that can be spent as an input in a new transaction. The key issue here is that a complete UTxO must be used as an input for a new transaction. UTxOs cannot be consumed in part. Rather, the difference is sent back to the source as a "change" during the transaction, which is represented as a new UTxO.
 
-For example, asuming zero transaction fees, let's say Alice has 100 ada and Bob has 20 ada. If Alice wants to transfer Bob 40 ada, Alice must consume the full 100 ada as input to the new transaction. This new transaction will have two outputs: 40 ada to Bob and 60 ada to Alice. 
+For example, assuming zero transaction fees, let's say Alice has 100 ada and Bob has 20 ada. If Alice wants to transfer 40 ada to Bob, Alice must consume the full 100 ada as input to the new transaction. This new transaction will have two outputs: 40 ada to Bob and 60 ada to Alice.
```

Comment:
```suggestion
For example, assuming zero transaction fees, let's say Alice has an address with a UTxO having 100 ada and Bob has an address with a UTxO having 20 ada. If Alice wants to transfer 40 ada to Bob, Alice must consume the full 100 ada as input to the new transaction. This new transaction will have two outputs: 40 ada to a new UTxO for Bob and 60 ada to a new UTxO for Alice.
```

Reply:
I kind of get it, but I agree, it might be too heavy... Not sure, maybe someone else can decide. :)

Reply:
I also think it's too heavy unless someone else can think of a more concise way of expressing it... I can't & I do a lot of this kind of editing 🤪